modules/setting/database.go

```diff
@@ -183,5 +189,11 @@ func ParseMSSQLHostPort(info string) (string, string) {
 	} else if len(info) > 0 {
 		host = info
 	}
+	if host == "" {
+		host = "127.0.0.1"
+	}
+	if port == "" {
+		port = "0"
```

Comment:
1433?

Reply:
There is another  `host, port := "127.0.0.1", "0"` above.

Either both change "0" to "1433", or both keep "0".

(personally I prefer to change both "0" to "1433", while keeping use "0" is also fine).

Reply:
yes

Reply:
I would like to point out that this is already the default value(see few lines up), so it should work accordingly otherwise we likely already should have received bug reports about it.

Reply:
Let's add a comment here to move on.